crates/daphne-worker/src/durable/aggregation_job_store.rs

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) 2025 Cloudflare, Inc. All rights reserved.
+// SPDX-License-Identifier: BSD-3-Clause
+
+use super::{req_parse, GcDurableObject};
+use crate::int_err;
+use daphne::messages::AggregationJobId;
+use daphne_service_utils::durable_requests::bindings::{
+    aggregation_job_store::{self, NewJobResponse},
+    DurableMethod,
+};
+use std::collections::HashSet;
+use worker::{js_sys::Uint8Array, Request, Response};
+
+super::mk_durable_object! {
+    struct AggregationJobStore {
+        state: State,
+        env: Env,
+        seen_agg_job_ids: Option<HashSet<AggregationJobId>>,
+    }
+}
+
+const SEEN_AGG_JOB_IDS_KEY: &str = "agg-job-ids";
+
+impl GcDurableObject for AggregationJobStore {
+    type DurableMethod = aggregation_job_store::Command;
+
+    fn with_state_and_env(state: worker::State, env: worker::Env) -> Self {
+        Self {
+            state,
+            env,
+            seen_agg_job_ids: None,
+        }
+    }
+
+    async fn handle(&mut self, mut req: Request) -> worker::Result<Response> {
+        match Self::DurableMethod::try_from_uri(&req.path()) {
+            Some(aggregation_job_store::Command::NewJob) => {
+                let aggregation_job_store::NewJobRequest { id, agg_job_hash } =
+                    req_parse(&mut req).await?;
+
+                let key = &id.to_string();
+                let response = match self.get::<Vec<u8>>(key).await? {
+                    Some(hash) if hash == *agg_job_hash => NewJobResponse::Ok,
+                    Some(_) => NewJobResponse::IllegalJobParameters,
+                    None => {
+                        self.state
+                            .storage()
+                            .put_raw(key, Uint8Array::from(agg_job_hash.as_ref()))
+                            .await?;
+                        let seen_agg_job_ids = self.load_seen_agg_job_ids().await?;
+                        seen_agg_job_ids.insert(id);
+                        self.store_seen_agg_job_ids().await?;
+                        NewJobResponse::Ok
+                    }
+                };
+
+                Response::from_json(&response)
+            }
+            Some(aggregation_job_store::Command::ListJobIds) => {
+                Response::from_json(&self.load_seen_agg_job_ids().await?)
+            }
+            None => Err(int_err(format!(
+                "AggregationJobStore: unexpected request: method={:?}; path={:?}",
+                req.method(),
+                req.path()
+            ))),
+        }
+    }
+
+    fn should_cleanup_at(&self) -> Option<worker::ScheduledTime> {
+        todo!()
+    }
```

Comment:
I noticed this was stubbed out and wanted to double check you didn't mean to fill it out in this PR